custom/care_pathways/sqldata.py

```diff
@@ -367,30 +380,49 @@ def columns(self):
                              CareCustomColumn('none', filters=self.filters + [RawFilter("maxmin = 0")])]),
         ]
 
-    def headers(self, data):
-        column_headers = []
-        groupped_headers = [list(v) for l,v in groupby(sorted(data.keys(), key=lambda x:x[2]), lambda x: x[2])]
-        for domain in groupped_headers:
-            groupped_practices = [list(v) for l,v in groupby(sorted(domain, key=lambda x:x[3]), lambda x: x[3])]
-            domain_group = DataTablesColumnGroup(self.group_name_fn(domain[0][2]))
-            for practice in groupped_practices:
-                domain_group.add_column(DataTablesColumn(self.group_name_fn(practice[0][3])))
+    @property
+    def domains_with_practices(self):
+        practices = self.config.get('practices')
+        if not practices:
+            practices = ('0', )
```

Comment:
actually i think `'0'` just a magic constant being used to check for `practices` and `domains` being empty? can you not replace that with just assigning them to `None` and checking below using `is None`?